pytools/graph.py

```diff
@@ -369,4 +370,45 @@ def compute_induced_subgraph(graph: Mapping[T, Set[T]],
 
 # }}}
 
+
+# {{{
+
+def get_graph_dot_code(graph: Mapping[T, Iterable[T]]) -> str:
```

Comment:
That type annotation does not make sense. `Iterables` are only guaranteed to allow iteration once. Did you mean `Sequence`?

Reply:
Sorry, I'm not sure I understand - is there a place in the function that requires multiple iteration on the values in `graph`? Also,  `Sequence` would disallow `set`s I think, so should it be a `Collection` instead?

Reply:
> Sorry, I'm not sure I understand - is there a place in the function that requires multiple iterations over the values in `graph` (I can only see the `for t in targets` line)?

I mean, if so, you'd be left with a dictionary full of exhausted iterators (e.g.).

> so should it be a `Collection` instead?

Yes, that's better. Thanks!

Reply:
Oh, I see! Should we change the signature of the other functions in this file that use Iterable to Collection also?

Reply:
Yes, I think that'd be better.

Reply:
Done in 93c12a5